### PR TITLE
adding info about tagging SIGs in pr wrangling guide

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Signed-off-by: RinkiyaKeDad <arshsharma461@gmail.com>

/sig docs
/language en
/cc @sftim @reylejano

### CODE REVIEW

File: content/en/docs/contribute/participate/pr-wranglers.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Each day in a week-long shift as PR Wrangler:
     - If you need to verify content, comment on the PR and request more details.
     - Assign relevant `sig/` label(s).
     - If needed, assign reviewers from the `reviewers:` block in the file's front matter.
+    - You can also tag a SIG for a review by commenting `@kubernetes/<sig>-pr-reviews` on the PR.
```
Comment (sftim) [inline content/en/docs/contribute/participate/pr-wranglers.md:29]: nit: This doesn't always work. GitHub autocomplete is a way to check though (you need to be an org member to use that autocompletion, I think).

All PR wranglers are org members.

**Comment by a-mccarthy** [comment]
/lgtm

My only thought is to add a link to a list of sigs, if we have one. I'm new to the whole review process, though, so i don't know if it is obvious which sig to request a review from by the time a PR wrangler is looking at a PR :)

**Comment by RinkiyaKeDad** [comment]
> add a link to a list of sigs

We could do that but I'm not sure if that would be of much help since for PR wranglers I think we can assume that they'd have the required context, WDYT? 🙂

**Comment by sftim** [comment]
:+1: for hyperlinking to list of SIGs, eg https://github.com/kubernetes/community/blob/master/sig-list.md

**Comment by RinkiyaKeDad** [comment]
Updates, PTAL :)

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**